apps/search/scripts/migrations/migration-utils.ts

```diff
@@ -1,20 +1,18 @@
 /* eslint-disable turbo/no-undeclared-env-vars */
-
-import { SaleorCloudAPL } from "@saleor/app-sdk/APL";
+import { aplType,saleorApp } from "../../saleor-app";
 
 export const verifyRequiredEnvs = () => {
+  if (aplType == "file") return;
+
   const requiredEnvs = ["REST_APL_TOKEN", "REST_APL_ENDPOINT"];
 
   if (!requiredEnvs.every((env) => process.env[env])) {
     throw new Error(`Missing envs: ${requiredEnvs.join(" | ")}`);
   }
 };
 
-export const fetchCloudAplEnvs = () => {
-  const saleorAPL = new SaleorCloudAPL({
-    token: process.env.REST_APL_TOKEN!,
-    resourceUrl: process.env.REST_APL_ENDPOINT!,
-  });
+export const fetchAplEnvs = () => {
+  const saleorAPL = saleorApp.apl;
```

Comment:
I think this change is fine, but I don't know how does it solve you problem? Are you running migrations locally?

Reply:
yeah, this is only for running it locally without making code changes, same as `if (aplType == "file") return;` - added both for future debugging.